deviceModeInit.js

```diff
@@ -231,12 +230,14 @@ var rudderTracking = (function () {
   function callProductListViewedEvent(productsOnThePage, productConsidered) {
     const elementsToSend = [];
     const productsToSend = [];
+    let excludeCurrentPage = false;
+    if (pagePathArr[pagePathArr.length - 2] == "products") {
+      excludeCurrentPage = true;
+    }
```

Comment:
Is it totally excluding sending a `product list viewed` event from the page with products in its URL?

Reply:
No this will stop only the main product on the product page to stop being included in product list view event